docs/Language_Naming_Format.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
```

Comment:
Consider putting this file under `docs/Developers` directory, which looks better to me

Reply:
@li-boxuan I have done the requested changes